test/UnitTests/Microsoft.Testing.Platform.UnitTests/Program.cs

```diff
@@ -19,8 +19,10 @@
 #if ENABLE_CODECOVERAGE
 builder.AddCodeCoverageProvider();
 #endif
-builder.AddCrashDumpProvider();
+builder.AddAppInsightsTelemetryProvider();
+builder.AddCrashDumpProvider(ignoreIfNotSupported: true);
 builder.AddHangDumpProvider();
+builder.AddRetryProvider();
```

Comment:
@Evangelink Do we want to use re-try for unit tests? I'm afraid this could hide race conditions and flaky tests that indicate real issues